Makefile.toml

```diff
@@ -1394,6 +1396,29 @@ rm -rf ${BUILDSYS_STATE_DIR}
 '''
 ]
 
+# This task will delete vendored Go code, primarily, the Go module cache.
+# The Go module cache is intentionally readonly and does not have writable
+# subdirectories or files. So, we first need to perform the `chmod` in order to
+# have permissions to delete it.
+# See for more context: https://github.com/golang/go/issues/27455
```

Comment:
I'm OK with linking these to the `clean` target, but a `purge` target might be better if there's no ordinary need to wipe out these caches. Downloading Go modules and crates again puts a load on upstream resources that I'd prefer to avoid as part of anyone's usual workflow.

Reply:
There is already a `purge-test` target which will clear all tests from a testsys cluster. So, creating a top level `purge` target felt abit weird since I'd be mixing concerns abit in the names. But that would work unless there's another idea?

Reply:
Perhaps something like `purge-cache`, if you're concerned with naming?